src/tests/intg/test_pam_responder.py

```diff
@@ -338,8 +338,11 @@ def simple_pam_cert_auth(request, passwd_ops_setup):
     conf = format_pam_cert_auth_conf(config)
     create_conf_fixture(request, conf)
     create_sssd_fixture(request)
+    passwd_ops_setup.useradd(**CANARY)
     passwd_ops_setup.useradd(**USER1)
     passwd_ops_setup.useradd(**USER2)
+    sync_files_provider()
```

Comment:
`useradd(**CANARY)` should be last before `sync_files_provider()`
But imo even better would be to provide name-to-synd as argument to `sync_files_provider()` and use last added name, thus avoiding "CANARY" user at all.

Reply:
Good catch. It's now only in the first patch, and I look for USER2 instead of CANARY.